spec/integration/kibana/spec.yml

```
@@ -65,6 +65,15 @@ spec:
       type: file
       contentMediaType: "application/json"
       pattern: '^.+\.json$'
+  - description: Folder containing security AI prompt assets
+    type: folder
+    name: security_ai_prompt
+    required: false
+    contents:
+    - description: A security AI prompt asset file
```

Are they specific to security or they could be used in other solutions?
```suggestion
  - description: Folder containing AI prompt assets
    type: folder
    name: ai_prompt
    required: false
    contents:
    - description: An AI prompt asset file
```

---

This is security only